lxc/network.go

```diff
@@ -157,6 +157,10 @@ func (c *cmdNetworkAttach) Run(cmd *cobra.Command, args []string) error {
 		device["nictype"] = "bridged"
 	}
 
+	if network.Type == "ipvlan" {
+		device["nictype"] = "ipvlan"
+	}
```

Comment:
Those changes to `lxc/network.go` aren't needed. The `network.Type` here refers to the parent host device as listed in `lxc network list`, there is no such thing as a `ipvlan` one of those.

Reply:
@stgraber thanks, will take them out.